meinberlin/apps/budgeting/api.py

```diff
@@ -85,12 +85,13 @@ def list(self, request, *args, **kwargs):
         ordering_choices = [('-created', _('Most recent')), ]
         if self.module.has_feature('rate', Proposal):
             ordering_choices += ('-positive_rating_count', _('Most popular')),
-        ordering_choices += ('-comment_count', _('Most commented')),
+        ordering_choices += ('-comment_count', _('Most commented')), \
+                            ('-daily_random', _('Random')),
```

Comment:
Why the -? It probably doesn't matter if we use the random ascending or descending.